python_modules/libraries/dagster-dg/docs/app/packages/[pkg]/page.tsx

```diff
@@ -0,0 +1,36 @@
+import PackageDetails from '@/app/components/PackageDetails';
+import getContents, {getPackage} from '@/util/getContents';
+
+import styles from './PackagePage.module.css';
+
+interface Props {
+  params: Promise<{pkg: string}>;
+}
+
+export default async function Page({params}: Props) {
+  const {pkg} = await params;
+  const contents = await getContents();
```

Comment:
same, We shouldn't be calling this inside the component. This should use `getStaticProps`. Otherwise you're sort of defeating the page-split bundling by requiring every page to fetch the entire JSON instead of just the slice of JSON needed for that page

Reply:
I think we need to add `import "server-only";` to this file to make sure we don't pull this in the client